CST-Group/meca
# Folders gradle

### Why was it necessary?

Some IDEs, such as Eclipse, were having a hard time being able to configure the Gradle Project within MECA because source folders were not following the Gradle Nature default scheme (src/main/java ...). To avoid such problems, we changed the src, test and resource folders to Gradle default scheme.

### How was it done?

src and test folders were moved to Gradle default scheme. `build.gradle` was changed in order to not point to different custom locations anymore.

### How to test?
Run a MECA project that you have pointing to the version of the commit hash `b76be867d26b6edd229dee600430a546a9dc6633`, like the dependency `com.github.CST-Group:meca:b76be867d26b6edd229dee600430a546a9dc6633` in the Gradle case, and check that everything works fine.

### Future works

No more next steps in this direction ;)
